packages/mongodb-dialects/java-driver/src/main/kotlin/com/mongodb/jbplugin/dialects/javadriver/glossary/JavaDriverDialectParser.kt

```diff
@@ -366,7 +366,7 @@ object JavaDriverDialectParser : DialectParser<PsiElement> {
                     HasValueReference(valueReference),
                 )
             )
-        } else if (method.parameterList.parameters.size == 2) {
+        } else if (method.parameterList.parameters.size == 2 && filter.argumentList.expressions.size == 2) {
```

Comment:
Found a bug here where the query was valid, but we detected the method should have 2 methods (because the method was properly imported in IntelliJ) but we didn't have both arguments supplied.